sql/connect/client/jdbc/src/main/scala/org/apache/spark/sql/connect/client/jdbc/SparkConnectDatabaseMetaData.scala

```diff
@@ -199,164 +162,114 @@ class SparkConnectDatabaseMetaData(conn: SparkConnectConnection) extends Databas
-  override def getMaxCharLiteralLength: Int =
-    throw new SQLFeatureNotSupportedException
+  override def getMaxCharLiteralLength: Int = 0
 
-  override def getMaxColumnNameLength: Int =
-    throw new SQLFeatureNotSupportedException
+  override def getMaxColumnNameLength: Int = 0
 
-  override def getMaxColumnsInGroupBy: Int =
-    throw new SQLFeatureNotSupportedException
+  override def getMaxColumnsInGroupBy: Int = 0
 
-  override def getMaxColumnsInIndex: Int =
-    throw new SQLFeatureNotSupportedException
+  override def getMaxColumnsInIndex: Int = 0
 
-  override def getMaxColumnsInOrderBy: Int =
-    throw new SQLFeatureNotSupportedException
+  override def getMaxColumnsInOrderBy: Int = 0
 
-  override def getMaxColumnsInSelect: Int =
-    throw new SQLFeatureNotSupportedException
+  override def getMaxColumnsInSelect: Int = 0
 
-  override def getMaxColumnsInTable: Int =
-    throw new SQLFeatureNotSupportedException
+  override def getMaxColumnsInTable: Int = 0
 
-  override def getMaxConnections: Int =
-    throw new SQLFeatureNotSupportedException
+  override def getMaxConnections: Int = 0
 
-  override def getMaxCursorNameLength: Int =
-    throw new SQLFeatureNotSupportedException
+  override def getMaxCursorNameLength: Int = 0
 
-  override def getMaxIndexLength: Int =
-    throw new SQLFeatureNotSupportedException
+  override def getMaxIndexLength: Int = 0
 
-  override def getMaxSchemaNameLength: Int =
-    throw new SQLFeatureNotSupportedException
+  override def getMaxSchemaNameLength: Int = 0
 
-  override def getMaxProcedureNameLength: Int =
-    throw new SQLFeatureNotSupportedException
+  override def getMaxProcedureNameLength: Int = 0
 
-  override def getMaxCatalogNameLength: Int =
-    throw new SQLFeatureNotSupportedException
+  override def getMaxCatalogNameLength: Int = 0
 
-  override def getMaxRowSize: Int =
-    throw new SQLFeatureNotSupportedException
+  override def getMaxRowSize: Int = 0
 
-  override def doesMaxRowSizeIncludeBlobs: Boolean =
-    throw new SQLFeatureNotSupportedException
+  override def doesMaxRowSizeIncludeBlobs: Boolean = false
 
-  override def getMaxStatementLength: Int =
-    throw new SQLFeatureNotSupportedException
+  override def getMaxStatementLength: Int = 0
 
-  override def getMaxStatements: Int =
-    throw new SQLFeatureNotSupportedException
+  override def getMaxStatements: Int = 0
 
-  override def getMaxTableNameLength: Int =
-    throw new SQLFeatureNotSupportedException
+  override def getMaxTableNameLength: Int = 0
 
-  override def getMaxTablesInSelect: Int =
-    throw new SQLFeatureNotSupportedException
+  override def getMaxTablesInSelect: Int = 0
 
-  override def getMaxUserNameLength: Int =
-    throw new SQLFeatureNotSupportedException
+  override def getMaxUserNameLength: Int = 0
 
-  override def getDefaultTransactionIsolation: Int =
-    throw new SQLFeatureNotSupportedException
+  override def getDefaultTransactionIsolation: Int = Connection.TRANSACTION_NONE
```

Comment:
Databricks JDBC v1.0.10-oss returns `TRANSACTION_READ_UNCOMMITTED`, I can't follow that, because `supportsTransactions` returns `false`